### DISCUSSION TITLE
[New to TE3] - is there a way to change the default layout to one of my own?

### DISCUSSION THREAD

**Discussion by bdn008** [TabularEditor/TabularEditor3#1378] [🙋♀️TE3 Q&A] [answered]
Hello TE3 community and support,

I am new to TE3 and loving it (now that I was able to get the enterprise version). I am going through the training and videos and find it extremely helpful, not to mention that I wish I had it a few years back when setup and debug was much more manual. but enough about me..

I am setting up a few layouts and I love the fact that I can.  However, I am trying to set one of my views as the default view and can't find a way to do that. Please advise

**Comment by otykier** [comment, marked as answer]
Welcome and thank you for the kind words :)

- If you want to override the Default layout, simply name your layout "Default" when saving it (using the **Window > Capture current layout...** menu option)
- Aside from that, there is not really a concept of "default" layout. Tabular Editor 3 will always restore the last layout used when the application is launched.

Hope this helps!

**Reply by bdn008** [reply]
Thanks for the quick answer. I was thinking that would be an option but didn't want to overwrite the initial default layout.

I am good with this solution but if I may make a suggestion, in some future release, please add a "Make default" option in the Manage layouts popup.

**Reply by otykier** [reply]
May I ask what the purpose of the "Make default" option would be? Are you thinking that you would like Tabular Editor to reset to one particular pre-defined layout every time it is launched, rather than restoring whatever layout was used in the last session?

**Reply by bdn008** [reply]
In a word, Yes.

I open different modules and want "my stuff"
